### PR TITLE
tests: check for errnos that sd_id128_get_machine actually returns

### LINKED ISSUE #118: Failing tests test_get_machine and test_get_machine_app_specific
While packaging this package (version 235) I get these two tests failing:
```
[   21s] =================================== FAILURES ===================================
[   21s] _______________________________ test_get_machine _______________________________
[   21s] 
[   21s]     def test_get_machine():
[   21s] >       u1 = id128.get_machine()
[   21s] E       FileNotFoundError: [Errno 2] No such file or directory
[   21s] 
[   21s] systemd/test/test_id128.py:24: FileNotFoundError
[   21s] ________________________ test_get_machine_app_specific _________________________
[   21s] 
[   21s]     def test_get_machine_app_specific():
[   21s]         a1 = uuid.uuid1()
[   21s]         a2 = uuid.uuid1()
[   21s]     
[   21s]         with skip_oserror(errno.ENOSYS):
[   21s] >           u1 = id128.get_machine_app_specific(a1)
[   21s] E           FileNotFoundError: [Errno 2] No such file or directory
[   21s] 
[   21s] systemd/test/test_id128.py:33: FileNotFoundError
[   21s] =========================== short test summary info ============================
[   21s] FAILED systemd/test/test_id128.py::test_get_machine - FileNotFoundError: [Err...
[   21s] FAILED systemd/test/test_id128.py::test_get_machine_app_specific - FileNotFou...
[   21s] ============ 2 failed, 64 passed, 2 skipped, 1 deselected in 0.29s =============
```
[Complete build log](https://github.com/systemd/python-systemd/files/9356800/_log.txt) with versions of all packages used and steps taken to reproduce.

FileNotFoundError could possibly be influenced by the limited building environment we build our packages in (something from `/dev/` or something like this?). Or are there some real files missing?

### PR DESCRIPTION
**Pull request by keszybz**
Fixes #118.

### CODE REVIEW

File: systemd/test/test_id128.py [hunk 1]
```diff
@@ -29,7 +29,8 @@ def test_get_machine_app_specific():
     a1 = uuid.uuid1()
     a2 = uuid.uuid1()
 
-    with skip_oserror(errno.ENOSYS):
+    # yikes, python2 doesn't know ENOMEDIUM
```
Comment (bluca) [inline systemd/test/test_id128.py:32]: Given the other PR, maybe merge it first and use enomedium here?

**Comment by mrc0mmand** [comment]
Since initializing `/etc/machine-id` for the tests to pass shouldn't be necessary anymore, can you drop https://github.com/systemd/python-systemd/blob/dc1dae2eaa00e2369c366f80f9bcd3f729b7abca/.github/workflows/install.yml#L79-L80 from the CI stuff?

File: systemd/test/test_id128.py (hunk 1 above)
Comment (keszybz) [reply systemd/test/test_id128.py:34]: I want this patch to be backportable without conflicts… This workaround an be dropped later.

**Review by bluca — APPROVED** [review]

**Comment by keszybz** [comment]
OK, the tests pass, and with the second commit, we know that we're testing without machine-id, so I'm pretty sure this resolved the original issue.

**Pull request merged**